portality/view/oaipmh.py

```diff
@@ -131,6 +148,9 @@ def make_resumption_token(metadata_prefix=None, from_date=None, until_date=None,
 class ResumptionTokenException(Exception):
     pass
 
+class SetSpecException(Exception):
+    pass
```

Comment:
I don't see a particular need to - all we'll be learning is that user's have fluffed their inputs, it doesn't really affect us.  They are not errors on our side - I mostly use these for message passing in the stack to affect the behaviour higher up.